### PR TITLE
worked on signin issue

### PR DESCRIPTION
**Pull request by shekhar456**
managed the state of login form

### CODE REVIEW

**Comment by sunuwarshekhar** [comment]
not clear

**Comment by shekhar456** [comment]
got it

**Review by shekhar456 — COMMENTED** [review]
nice

**Review by sunuwarshekhar — COMMENTED** [review]
not bad

File: src/auth/Login.jsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ export default function Login() {
           >
             <Avatar sx={{ m: 1, bgcolor: "secondary.main" }}></Avatar>
             <Typography component="h1" variant="h5">
-              Sign in
+              SignIn CarBidding
```
Comment (sunuwarshekhar) [inline src/auth/Login.jsx:120]: explain abit more

**Review by sunuwarshekhar — COMMENTED** [review]
nice

File: src/auth/Login.jsx (hunk 1 above)
Comment (sunuwarshekhar) [inline src/auth/Login.jsx:120]: add site to it

Comment (shekhar456) [reply src/auth/Login.jsx:120]: sure

Comment (shekhar456) [reply src/auth/Login.jsx:120]: ok

**Review by sunuwarshekhar — APPROVED** [review]
good work

**Review by sunuwarshekhar — APPROVED** [review]

**Pull request merged**